frontend/src/app/session-overview/session-overview.component.ts

```diff
@@ -41,10 +43,20 @@ export class SessionOverviewComponent implements OnInit {
       .getCurrentSessions()
       .subscribe((res: Array<Session>) => {
         this.sessions = res;
+        for (const id in this.deletionFormGroup.controls) {
+          this.deletionFormGroup.removeControl(id);
+        }
```

Comment:
In which case we would get into this for loop?

Reply:
Usually, the `deletionFormGroup` is always empty, because `refereshSessions()` gets only called once in `ngOnInit()`. But to prevent some side effects in the future, e.g. if there is a refresh button, I would keep the code block.